packages/flutter_tools/lib/src/plugins.dart

```diff
@@ -265,29 +292,52 @@ List<Plugin> findPlugins(FlutterProject project) {
   return plugins;
 }
 
-/// Returns true if .flutter-plugins has changed, otherwise returns false.
+/// Returns true if .flutter-plugins or .flutter-plugins-dependencies have changed,
```

Comment:
Ultra-nit, // instead of ///

It's not a full dart doc since it only partially describes this function (its return value). If ever this becomes a public method, it's up to that person to write a proper dart doc

Reply:
`///` makes the comment show up in the IDE.